OrbitService/LinuxTracingGrpcHandler.h

```diff
@@ -5,23 +5,21 @@
 #ifndef ORBIT_SERVICE_LINUX_TRACING_GRPC_HANDLER_H_
 #define ORBIT_SERVICE_LINUX_TRACING_GRPC_HANDLER_H_
 
-#include <OrbitBase/Logging.h>
-#include <OrbitBase/Tracing.h>
-#include <OrbitLinuxTracing/Tracer.h>
-#include <OrbitLinuxTracing/TracerListener.h>
-
+#include "CaptureResponseListener.h"
+#include "OrbitBase/Logging.h"
+#include "OrbitBase/Tracing.h"
+#include "OrbitLinuxTracing/Tracer.h"
+#include "OrbitLinuxTracing/TracerListener.h"
 #include "absl/container/flat_hash_set.h"
 #include "absl/synchronization/mutex.h"
-#include "services.grpc.pb.h"
+#include "capture.pb.h"
 
 namespace orbit_service {
 
 class LinuxTracingGrpcHandler : public LinuxTracing::TracerListener {
```

Comment:
Should this just be renamed (back) to `LinuxTracingHandler` then, as it's independent from gRPC now?

Reply:
Yeah, I was thinking about this and then forgot - done.